dissect/target/helpers/mount.py

```diff
@@ -100,7 +118,18 @@ def readlink(self, path: str) -> str:
             raise FuseOSError(errno.EIO)
 
     def release(self, path: str, fh: int) -> int:
+        if file := self.file_handles.get(fh):
+            file.close()
+
         del self.file_handles[fh]
+        return 0
 
     def releasedir(self, path: str, fh: int) -> int:
         del self.dir_handles[fh]
+        return 0
+
+    if feature_enabled(Feature.BETA):
```

Comment:
Doesn't really make sense to guard this function behind the beta flag because it relies on FUSE3, not a beta feature. Better use something like a `HAS_FUSE3` flag that you define in the import.